lib/hooks.js

```diff
@@ -61,6 +64,14 @@ function assignLifeCycleHooksToContext (context, hooks) {
   return Object.assign(context, contextHooks)
 }
 
+function assignApplicationHooksToContext (context, hooks) {
+  const contextHooks = {
+    onExtendSchema: null
+  }
+  if (hooks.onExtendSchema.length > 0) contextHooks.onExtendSchema = hooks.onExtendSchema.slice()
```

Comment:
formatting: prefer wrapping conditional bodies in brackets for readability

Reply:
I used the same style of:

https://github.com/mercurius-js/mercurius/blob/6099ac590e4894367ffce3459b32c3199dc6babd/lib/hooks.js#L45-L65